src/pages/Accounts/AccountSettings/MergeAccountShellsScreen.tsx

```diff
@@ -88,6 +91,9 @@ const styles = StyleSheet.create( {
     position: 'absolute',
     bottom: 30,
     alignSelf: 'center',
+    shadowColor: Colors.shadowBlue,
+    shadowOpacity: 1,
+    shadowOffset: { width: 15, height: 15 },
   },
```

Comment:
@cakesoft-nikhita  proceedButtonContainer seems like a good candidate to be extracted out as a component.
is this feasible?